docs/openstack-openedx/supported-deployment.md

```diff
@@ -0,0 +1,99 @@
+1. Set up MongoDB: Create two databases - one for courseware and one for forum posts. Create a separate username and password for each database.
+1. Set up RabbitMQ: Create a username+password for "celery" and another username+password for "xqueue".
+1. Set up ElasticSearch as listed in the requirements section above.
+1. Create a private git repository (we recommend using GitHub or GitLab) with a name like "sitename-vars" where sitename is the name of your new Open edX site
+1. In that new repository, create a vars.yml file using [this template](https://github.com/open-craft/opencraft/pull/204) as a template
+1. Go through `vars.yml` and customize each variable to match your planned deployment. The comments included in the file offer an explanation of what each setting does.
+1. Set up a "director" VM following [these instructions](../shared/director/).
+1. Run ansible to provision the instance using [the edx-stateless.yml playbook](https://github.com/edx/configuration/blob/master/playbooks/edx-stateless.yml):
+   ```
+   ansible-playbook -i 1.2.3.4 -e@../path/to/sitename-vars.yml -u ubuntu edx-stateless.yml
+   ```
+   Where 1.2.3.4 is the IP of the VM provisioned in step 1.
+1. Update /etc/hosts to point to the new instance, and test it
+1. Update the DNS or OpenStack LBaaS to point to the new VM
+1. Create a user account
+1. SSH in to the instance and give that account admin permissions
+   1. `sudo -Hu edxapp bash`
+   1. `cd && . edxapp_env  && . ./venvs/edxapp/bin/activate && cd edx-platform/`
+   1. `from django.contribu.auth.models import User`
+   1. `u = User.objects.get(email='your_email@example.com')`
+   1. `u.is_staff = True`
+   1. `u.is_superuser = True`
+   1. `u.save()`
+1. What now? See http://docs.edx.org/ for documentation about configuration and usage of your new instance.
+
+
+## How to deploy an update
+1. Provision a brand new OpenStack VM similar to the VM used for the initial deployment.
+1. Update `sitename-vars.yml` as desired
+   1. For example, if you are upgrading to a new named release of Open edX, change the `OPENEDX_RELEASE` variable from "open-release/ficus.master" to the git branch name of the new release (e.g. "open-release/ginkgo.master", "open-release/hawthorn.master" etc.)
+1. `cd ~/configuration/playbooks`
+1. `source ~/mysite-deploy-ansible/bin/activate`
+1. Run ansible to provision the instance using the edx-stateless.yml playbook:
+   ```
+   ansible-playbook -i 1.2.3.4 -e@../path/to/sitename-vars.yml -u ubuntu edx-stateless.yml
+   ```
+   Where 1.2.3.4 is the IP of the VM provisioned in step 1.
+1. Update `/etc/hosts` to point to the new instance, and test it
+1. Update the DNS or OpenStack LBaaS to point to the new VM instead of the old one.
+ 
+
+## How to get support (Proposal - this is not yet actively supported)
```

Comment:
@bradenmacdonald I merged and deployed: http://opencraft.com/doc/openedx-deployment/

But reading it again there, I've noticed there are still mentions of supporting it, so I'm removing them, see https://github.com/open-craft/openedx-deployment/commit/5a8e64dbfd98abb2a00f2cd170f48e72d5981df4